src/polyswarmd/eth.py

```diff
@@ -151,6 +160,33 @@ def build_transaction(call, nonce):
     return call.buildTransaction(options)
 
 
+def is_withdrawal(tx):
+    """
+    Take a transaction and return True if that transaction is a withdrawal
+    """
+    error = True
+    function_hash = g.chain.w3.toHex(tx.data[:4])
+    if len(tx.data) != 68 or function_hash != transfer_signature_hash:
```

Comment:
instead of using literal lengths and offsets below, can we decode the func args with decode_abi (example in getTransactionError)